blocks/library/button/index.js

```diff
@@ -49,22 +50,44 @@ registerBlockType( 'core/button', {
 	},
 
 	getEditWrapperProps( attributes ) {
-		const { align } = attributes;
+		const { align, clear } = attributes;
+		const props = {};
+
 		if ( 'left' === align || 'right' === align || 'center' === align ) {
-			return { 'data-align': align };
+			props[ 'data-align' ] = align;
+		}
+
+		if ( clear ) {
+			props[ 'data-clear' ] = 'true';
 		}
+
+		return props;
 	},
 
 	edit( { attributes, setAttributes, focus, setFocus, className } ) {
-		const { text, url, title, align, color } = attributes;
+		const { text, url, title, align, color, clear } = attributes;
 		const updateAlignment = ( nextAlign ) => setAttributes( { align: nextAlign } );
+		const toggleClear = () => setAttributes( { clear: ! clear } );
+
+		const clearControl = (
+			<ToggleControl
+				label={ __( 'Stand on a line' ) }
+				checked={ !! clear }
+				onChange={ toggleClear }
+			/>
+		);
 
 		return [
 			focus && (
 				<BlockControls key="controls">
 					<BlockAlignmentToolbar value={ align } onChange={ updateAlignment } />
 				</BlockControls>
 			),
+			focus && (
+				<InspectorControls key="inspector">
+					{clearControl}
```

Comment:
@truongwp one more little thing before we merge. Can you add spaces around `clearControl` to satisfy WordPress coding guidelines? I missed that yesterday, thanks!